docs_src/microservices/core/metadata/details/SpecialChar.md

```diff
@@ -0,0 +1,45 @@
+---
+title: Core Metadata - Special Character
+---
+
+# Core Metadata - Special Character
+
+Users sometimes assign device names or other metadata values that include special characters. When these values are updated or processed (such as through APIs or configuration files), certain characters must be escaped to ensure correct parsing and to prevent unexpected behavior.
+
+Failing to escape these characters can lead to errors, misinterpretation of metadata, or failed requests. Therefore, it's important to identify and properly handle special characters when entering or modifying metadata values.
```

Comment:
to be clear, the escape mechanism is URL encoding, and the reference has to be provided